### PR TITLE
BoS capes

### PR DESCRIPTION
**Pull request by Tamumus**
## About The Pull Request

Add capes to the elders and paladins (Courtesy of Asterix!),they spawn with them on their necks. paladins capes acts as shoulder holsters.

## Why It's Good For The Game

in the words of our wise LM and headknight "bitches love capes".

## Changelog
:cl:
add: Added capes for paladins
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -50,20 +50,26 @@
 	desc = "This is the overseers cloak.  Issued by the Vault-tec corporation to easily identify the overseer. This cloak has been passed down from overseer to overseer"
 	icon_state = "overseercloak"
 
-/obj/item/clothing/neck/cloak/bos/left
+/obj/item/clothin/neck/cloak/bos
 	name = "Brotherhood of Steel shoulder cape"
-	desc = "Issued to the Elders of the Brotherhood. Style over substance is important. This one is designed to be worn over the shoulder."
+	desc = "Issued to the Elders of the Brotherhood. Style over substance is important.This one is designed to be worn over the shoulder."
```
Comment (BurpleBineapple) [inline code/modules/clothing/suits/cloaks.dm:55]:
```suggestion
	desc = "Issued to the Elders of the Brotherhood. Style over substance is important. This one is designed to be worn over the shoulder."
```
Grammar memes

File: code/modules/clothing/suits/cloaks.dm [hunk 2]
```diff
@@ -50,20 +50,26 @@
...
+	icon_state = "bosshouldercape_l"
+
+/obj/item/clothin/neck/cloak/bos/left
+	name = "Brotherhood of Steel shoulder cape"
+	desc = "Issued to the Elders of the Brotherhood. Style over substance is important.This one is designed to be worn over the shoulder."
```
Comment (BurpleBineapple) [inline code/modules/clothing/suits/cloaks.dm:60]:
```suggestion
	desc = "Issued to the Elders of the Brotherhood. Style over substance is important. This one is designed to be worn over the shoulder."
```

File: code/modules/clothing/suits/cloaks.dm [hunk 3]
```diff
@@ -50,20 +50,26 @@
 	desc = "This is the overseers cloak.  Issued by the Vault-tec corporation to easily identify the overseer. This cloak has been passed down from overseer to overseer"
 	icon_state = "overseercloak"
 
-/obj/item/clothing/neck/cloak/bos/left
+/obj/item/clothin/neck/cloak/bos
```
Comment (BurpleBineapple) [inline code/modules/clothing/suits/cloaks.dm:53]:
```suggestion
/obj/item/clothing/neck/cloak/bos
```

**Review by BurpleBineapple — CHANGES_REQUESTED** [review]
A few minor errors with spelling and grammar. One incorrect item path as well

**Comment by BurpleBineapple** [comment]
Also, about the holster. From what I can see this will allow BoS cloaks to hold ammo boxes? All holsters were barred from this during the rebase. Adding this to the Brotherhood cloak-holsters seems like a bad idea.

File: code/modules/clothing/suits/cloaks.dm (hunk 1 above)
Comment (Tamumus) [reply code/modules/clothing/suits/cloaks.dm]: fixed, thanks for catching it.

File: code/modules/clothing/suits/cloaks.dm (hunk 2 above)
Comment (Tamumus) [reply code/modules/clothing/suits/cloaks.dm]: fixed as well.

File: code/modules/clothing/suits/cloaks.dm (hunk 3 above)
Comment (Tamumus) [reply code/modules/clothing/suits/cloaks.dm:53]: oh,lord. good catch.

**Comment by Tamumus** [comment]
> Also, about the holster. From what I can see this will allow BoS cloaks to hold ammo boxes? All holsters were barred from this during the rebase. Adding this to the Brotherhood cloak-holsters seems like a bad idea.

It's exactly the same as a shoulder holster. The /obj/item/ammo_box/.. is used for storing..magazines. I tested it extensively locally just in case.

**Pull request merged**